soc/arm/bcm_vk/viper/mmu_regions.c

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2020 Broadcom
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <soc.h>
+#include <arch/arm/aarch64/arm_mmu.h>
+
+#define SZ_4K	0x1000
+#define SZ_1M	(256 * SZ_4K)
+#define SZ_2M	(2 * SZ_1M)
+#define SZ_4M	(2 * SZ_2M)
+#define SZ_512M (256 * SZ_2M)
+#define SZ_1G   (2 * SZ_512M)
```

Comment:
I wonder if we should add these in `arch/arm/aarch64/arm_mmu.h`, since it applies to all AArch64 devices.

Reply:
Yes, makes sense to add there.